inference-engine/tests/functional/plugin/cpu/single_layer_tests/depth_to_space.cpp

```diff
@@ -2,56 +2,90 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <shared_test_classes/single_layer/depth_to_space.hpp>
+#include "shared_test_classes/single_layer/depth_to_space.hpp"
 #include "test_utils/cpu_test_utils.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
 
 using namespace InferenceEngine;
 using namespace CPUTestUtils;
 using namespace ngraph::opset3;
+using namespace ov::test;
 
 namespace CPULayerTestsDefinitions {
 
-typedef std::tuple<
-        LayerTestsDefinitions::depthToSpaceParamsTuple,
+using DepthToSpaceInputShapes = std::pair<std::vector<ov::PartialShape>, std::vector<ov::Shape>>;
```

Comment:
There is `ov::test::InputShape` and the new paradigm is to use std::vector<ov::test::InputShape> with InputShape per each input